proactive/ProactiveGateway.py

```diff
@@ -162,8 +162,21 @@ def getJobResult(self, job_id, timeout=60000):
         return '\n'.join(v for v in all_results)
 
     def getTaskResult(self, job_id, task_name, timeout=60000):
+        task_result = self.proactive_scheduler_client.waitForTask(str(job_id), task_name, timeout)
+        if type(task_result.getValue()) is bytes:
+            task_result = task_result.getValue().decode('ascii')
```

Comment:
Maybe you should create a method that will encapsulate `decode('ascii')`. It may help in the future if `ascii` is not used to decode anymore.

Reply:
I will do that. Thanks!